smoke-tests/src/test/groovy/io/opentelemetry/smoketest/WildflySmokeTest.groovy

```diff
@@ -41,17 +41,6 @@ abstract class WildflySmokeTest extends AppServerTest {
   }
 }
 
-abstract class WildflyJdk8SmokeTest extends WildflySmokeTest {
```

Comment:
when updating the wildfly versions I unintentionally removed this super class from wildfly 8 test, as they still pass I guess it is fine and this is not needed any more